lib/jekyll/admin/server/static_file.rb

```diff
@@ -40,7 +43,21 @@ def static_files
         end
 
         def static_file
-          static_files.find { |f| f.path == static_file_path }
+          file = static_files.find { |f| f.path == static_file_path }
+          if !file
+            file_list = static_files.select do |f|
```

Comment:
I'd break `file_list` into it's own method. `static_file`'s getting a bit hard to read.